app/forms/idv/doc_pii_form.rb

```diff
@@ -0,0 +1,51 @@
+module Idv
+  class DocPiiForm
+    include ActiveModel::Model
+
+    validates_presence_of :first_name
+    validates_presence_of :last_name
+    validates_presence_of :dob
+
+    attr_reader :first_name, :last_name, :dob
+
+    def initialize(pii)
+      @first_name = pii[:first_name]
+      @last_name = pii[:last_name]
+      @dob = pii[:dob]
+    end
+
+    def submit
+      FormResponse.new(
+        success: valid?,
+        errors: error_hash,
+        extra: { original_errors: errors },
+      )
+    end
+
+    private
+
+    def error_hash
+      error_count = [name_error, dob_error].count(&:present?)
+
+      return {} if error_count.zero?
+      return { pii: multiple_errors_message } if error_count > 1
+      { pii: name_error || dob_error || state_id_error }
```

Comment:
the count bit is throwing me....since we only have two of these, what about expanding out some if statements?

Also how is `state_id_error` ever returned? If neither error is present we return empty?

```suggestion
      if name_error.present? && dob_error.present?
        { pii: multiple_errors_message } 
      elsif name_error.present? || dob_error.present?
        { pii: name_error || dob_error }
      else
        {}
      end
```

Reply:
On second inspection, there's no such thing as `state_id_error`..... good thing Ruby evaluates super lazily lol

Reply:
Oops, something that should have been removed